firebase.go

```diff
@@ -128,8 +145,43 @@ func NewApp(ctx context.Context, config *Config, opts ...option.ClientOption) (*
 
 	return &App{
 		creds:         creds,
+		databaseURL:   config.DatabaseURL,
 		projectID:     pid,
 		storageBucket: config.StorageBucket,
 		opts:          o,
 	}, nil
 }
+
+func amendDefaultConfig(config *Config) (*Config, error) {
+	fbc := &Config{}
+	confFileName := os.Getenv(FirebaseEnvName)
+	if len(confFileName) == 0 {
+		return config, nil
+	}
+	dat, err := ioutil.ReadFile(confFileName)
+	if err != nil {
+		return nil, err
+	}
+	err = json.Unmarshal(dat, fbc)
+	if err != nil {
+		return nil, err
+	}
+	jsonData := map[string]string{}
+	json.Unmarshal(dat, &jsonData)
```

Comment:
Please add a TODO that this code can be removed after we support Go 1.10+, by changing the previous Unmarshal to something like:

	d := json.NewDecoder(bytes.NewReader(dat))
	d.DisallowUnknownFields()
	e := d.Decode(fbc)

https://github.com/golang/go/commit/2596a0c075aeddec571cd658f748ac7a712a2b69